.travis.yml

```diff
@@ -14,32 +14,31 @@ matrix:
 
   include:
     - php: 5.5.9
-      env: LARAVEL="5.1.*"
+      env: LARAVEL="5.1.*" LUMEN="5.1.*"
     - php: hhvm-3.18
-      env: LARAVEL="5.1.*"
+      env: LARAVEL="5.1.*" LUMEN="5.1.*"
     - php: 7.0
-      env: LARAVEL="5.5.*"
+      env: LARAVEL="5.5.*" LUMEN="5.5.*"
     - php: 7.1
-      env: LARAVEL="^5.5"
+      env: LARAVEL="^5.5" LUMEN="^5.5"
     - php: 7.2
-      env: LARAVEL="^5.5"
+      env: LARAVEL="^5.5" LUMEN="^5.5"
     - php: 7.2
-      env: LARAVEL="^6.0"
+      env: LARAVEL="^6.0" LUMEN="^6.0"
     - php: 7.2
-      env: LARAVEL="^7.0"
+      env: LARAVEL="^7.0" LUMEN="7.x-dev"
```

Comment:
`^7.0@dev` is the version constraint you want. `7.x-dev` is actually not a version constraint.

Reply:
wonderful